### DISCUSSION TITLE
What is a sensible maximum number of open connections for a single Node.js process?

### DISCUSSION THREAD

**Discussion by Ethan-Arrowood** [nodejs/undici#1233] [Q&A] [answered]
What is a sensible maximum number of open connections for a Node.js process that is using Undici's Agent/Pool client to handle HTTP networking?

**Comment by mcollina** [comment, marked as answer]
This is totally dependent on the Operating System and its configuration. Your main limit is the number of file descriptors.

**Reply by Ethan-Arrowood** [reply]
Thank you!
